--- perl_checker-1.2.24/src/test/test_it.orig	2013-04-10 21:25:52.335861264 +0100
+++ perl_checker-1.2.24/src/test/test_it	2013-04-10 21:42:12.365804003 +0100
@@ -84,6 +84,7 @@
     }
     output('.pl', @l);
     my @raw_log = `../perl_checker .pl`;
+    print join("\n", @raw_log);
     die "@raw_log in .pl ($file):\n" . join('', @{$test->{lines}}) if any { /^syntax error$/ } @raw_log;
 
     my $f;
